nifi-commons/nifi-property-protection-azure/pom.xml

```diff
@@ -97,6 +97,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.woodstox</groupId>
+            <artifactId>woodstox-core</artifactId>
+            <version>5.0.3</version>
+            <!-- This appears to be the common version in NiFi -->
+        </dependency>
```

Comment:
The current Maven configuration has exclusions for `woodstox-core` to avoid unexpected runtime behavior with a different XML implementation. The `jackson-dataformat-xml` dependency includes a dependency on `woodstox-core` that has a major version of 6, instead of 5.